contracts/core/EntryPoint.sol

```diff
@@ -50,7 +50,10 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
     bytes32 private constant INNER_REVERT_LOW_PREFUND = hex"deadaa51";
 
     uint256 private constant REVERT_REASON_MAX_LEN = 2048;
+    // Penalty charged for either unused execution gas or postOp gas
     uint256 private constant PENALTY_PERCENT = 10;
+    // Threshold below which no penalty would be charged
+    uint256 private constant PENALTY_GAS_THRESHOLD = 4e4;
```

Comment:
1. don't use exponential notation in solidity code. better 40_000
2. what is the rationale for this value?